src/Categories/Category/Construction/Comma.agda

```diff
@@ -139,3 +139,11 @@ module _ {C : Category o₁ ℓ₁ e₁} {D : Category o₂ ℓ₂ e₂} where
 
   _↘_ : (S : Functor D C) (X : C.Obj) → Category _ _ _
   S ↘ X = S ↓ const! X
+
+module _ (C : Category o₁ ℓ₁ e₁) where
+  private
+    module C = Category C
+
+  -- The slice, or over category
+  _/_ : (X : C.Obj) → Category _ _ _
```

Comment:
there is a standalone slice category. is this preferable? where is it used?

Reply:
Ah, I was looking in `Categories.Category.Construction` for this. We should probably move `Slice` in there at some point. Either way, I've updated the code to use `Slice` instead.